avm/src/vm/deploy.rs

```diff
@@ -107,13 +107,8 @@ pub fn kill(app_id: &str) {
   });
   let resp = post(format!("{}/v1/kill", URL), body);
   match resp {
-    Ok(_) => {
-      println!("Killing app with id {} if it exists...\n", app_id);
-      status();
```

Comment:
You don't need these status calls anymore?

Reply:
They should have never been there in part because AWS doesn't pick up the change kickly enough so it wasn't showing the proper status of the kill-ed / new one